### DISCUSSION TITLE
pangeo-2.0

### DISCUSSION THREAD

**Discussion by giscus** [mrocklin/mrocklin.github.io#21] [Giscus]
# pangeo-2.0

Past: In January 2018 I published a blogpost titled Pangeo: JupyterHub, Dask, and XArray on the Cloud, which introduced a new architecture for running scalable Python computations on the cloud usin...

https://matthewrocklin.com/pangeo-2.0.html

**Comment by bstadlbauer** [comment]
Hi Matthew!

Thanks for the write-up! I've tried to reach out, but got a reply that pangeo-2.0-discovery@coiled.io does not exist. I've reached out on your Coiled email (found that on the Coiled Slack) instead.

**Reply by mrocklin** [reply]
That's strange!  I've been getting responses from other people on my e-mail (I don't use slack much, so not surprised if something there is off).  I'm curious what's causing the issue.  I'm also not getting an e-mail from you on my corporate e-mail.  🤷 

Len also reached out and sang your praises in his note 🙂 

I'm at my github handle at coiled.io if that's helpful.  I've also reached out to a gmx e-mail address of yours that I found in the flyte git repo.

**Reply by bstadlbauer** [reply]
I've done some investigation and it seems like my email provider (gmx.net) seems to have issues, sorry for the inconvenience there! I've scheduled something for later today, looking forward to chatting!

**Comment by danielmcquillen** [comment]
Really helpful article! Thanks for posting this has offered a lot of pointers to good info.

I'm trying to find a good platform for hosting Jupyter notebooks within an e-Learning course (within an open-source LMS I wrote). We want to provide compute-intensive learning material ... e.g. domain-specific jupyter notebooks ... as part of course contents.

Usually the way that's done is pulling a remote tool (like one that hosts Jupyter notebooks) into a course in an iframe via the "LTIv1.3" connection protocol. LTIv1.3 is just a fancy Open ID connect mechanism with some extra info for e-Learning.

Zero to JupyterHub has a plugin already that supports this (LTIAuthenticator) but I'm finding it hard to conceptualize and implement a way to spin up and manage notebooks for arbitrary students in arbitrary courses using things like Dockerfiles in GitHub repos for the notebook definitions. It probably can be done but I can't see it yet because I'm a newb.

So wondering if doing something like this would be easier using Nebari? Or perhaps another platform?

It could be I should just ditch the whole LTI thing, and just have the LMS itself manage spinning up notebooks and providing access to students. In that case any recommendations?

Thanks for any thoughts, Matthew.
